scripts/EndOfRunMonitor/ISISendOfRunMonitor.py

```diff
@@ -24,7 +24,8 @@
          facilityCycle.endDate"
 
 TIME_CONSTANT = 1  # Time between file reads (in seconds)
-DEBUG = False
+DEBUG = False  # If True will check fake_archive folder for the last_run.txt file and will not send data to DataReady queue"
```

Comment:
Thanks for this comment also and for adding fake_archive to the path further down which makes this clearer. Sometimes a word like DEBUG is overused, in this case, now reading your comment, I would suggest renaming it to USE_FAKE_ARVHIVE_FOLDER, while still leave your comment, to be more specific.

Reply:
Do you think calling it USE_FAKE_ARCHIVE_FOLDER is best since if it's true it also means messages aren't sent to the DataReady queue? Calling it USE_FAKE_ARCHIVE_FOLDER gives the impression that that's its only use.
I could create another variable called SEND_TO_DATA_READY which could be useful for testing too?

Reply:
Perhaps SEND_TO_DATA_READY is a better option - I will let you decide. My main point is that where possible it is better to use more specific/descriptive names for variables. DEBUG is prone to be interpreted differently by different developers so if can be more specific is a bonus. A generic DEBUG option would be better placed in the settings file